### PR TITLE
doc/rados: edit user-management.rst (1 of x)

### PR DESCRIPTION
**Pull request by zdover23**
Edit doc/rados/operations/user-management.rst. (1 of x)

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/user-management.rst [hunk 1]
```diff
@@ -45,118 +48,120 @@ Ceph interprets the command like this:
...
 
-Sometimes Ceph's user type may seem confusing, because the Ceph command line
+Sometimes Ceph's user type might seem confusing, because the Ceph command line
 allows you to specify a user with or without the type, depending upon your
 command line usage. If you specify ``--user`` or ``--id``, you can omit the
-type. So ``client.user1`` can be entered simply as ``user1``. If you specify
-``--name`` or ``-n``, you must specify the type and name, such as
-``client.user1``. We recommend using the type and name as a best practice
-wherever possible.
+type. For example, ``client.user1`` can be entered simply as ``user1``. On the
```
Comment (anthonyeleven) [inline doc/rados/operations/user-management.rst:84]: Suggest above s/can/may/ twice.

Above Below "specify" is used twice, suggest:

>When specifying \`\`--user\`\` or \`\`--id\`\`, you may omit the type, entering \`\`client.user1\`\` as simply \`\`user1\`\`.  However, when specifying  \`\`--name`\` or ``-n\``, you must supply both the type and the name, for example: `\`client.user1\``.  We recommend supplying both the type and the name as a best practice whenever possible.

**Review by anthonyeleven — APPROVED** [review]
Several suggestions proffered.
Commit and PR titles should be prefixed with `doc/rados/operations:`

File: doc/rados/operations/user-management.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/operations/user-management.rst:84]: "supplying" is good.

File: doc/rados/operations/user-management.rst [hunk 2]
```diff
@@ -45,118 +48,120 @@ Ceph interprets the command like this:
...
+the type will always be ``client``. Ceph identifies users in a "period-
+delimited form" that consists of the user type and the user ID: for example,
 ``TYPE.ID``, ``client.admin``, or ``client.user1``. The reason for user typing
-is that Ceph Monitors, OSDs, and Metadata Servers also use the Cephx protocol,
-but they are not clients. Distinguishing the user type helps to distinguish
-between client users and other users--streamlining access control, user
-monitoring and traceability.
+is that the Cephx protocol is used not only by clients but also non-clients,
+such as Ceph Monitors, OSDs, and Metadata Servers. Distinguishing the user type
+helps to distinguish between client users and other users--and this distinction
```
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/rados/operations/user-management.rst:78]: We agonized over this sentence for about ten minutes. I'll chop it up.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/50640 - Reef backport
https://github.com/ceph/ceph/pull/50641 - Quincy backport
https://github.com/ceph/ceph/pull/50642 - Pacific backport

**Pull request merged**